core/services/flux_monitor.go

```diff
@@ -73,12 +74,20 @@ func (fm *concreteFluxMonitor) Start() error {
 
 	go fm.actionConsumer(fm.ctx)
 
-	var merr error
+	count := 0
+	errChan := make(chan error)
 	err := fm.store.Jobs(func(j *models.JobSpec) bool {
-		err := fm.AddJob(*j)
-		merr = multierr.Combine(merr, err)
+		go func(j *models.JobSpec) {
+			errChan <- fm.AddJob(*j)
```

Comment:
`errChan` has no buffer, and is never read from, what is it for?

Reply:
It was for the case mentioned above, thanks for catching this one.

This code should mirror what was implemented before e1d9a36